packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/Choice.js

```diff
@@ -0,0 +1,82 @@
+import { Box, Button, Carousel, Heading, Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { GoldButton, Media } from '@zooniverse/react-components'
```

Comment:
`GoldButton` is an older implementation that I keep meaning to deprecate. Use `PrimaryButton` instead with the gold theme (I think that's default).